examples/order_review_prediction/README.md

```diff
@@ -1,10 +1,10 @@
-# Predicting Order Reviews in dbt DAG
+# Predicting Order Review Scores in dbt DAG
 
-In this e-commerce example walk through, we will train a machine learning model to predict order reviews in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
+In this e-commerce example walk through, we will train a machine learning model to predict order review scores in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
 
-We will use `layer.automl()` functionality to train an ML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
+We will use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
 
-This is all the SQL we need to train our ML model within the dbt DAG. You just pass Layer the model type, input features and the target column we want to predict:
+This is all the SQL you need to train your ML model within the dbt DAG. You just pass Layer the model type, input features and the target column we want to predict:
```

Comment:
(changing person) If we want to use `you` here, we should change all previous steps from `we` to `you` as well.